CollapseLauncher/NativeMethods.txt

```diff
@@ -0,0 +1 @@
+CoCreateInstance
```

Comment:
Why does this exist?

Reply:
It's for listing the [NativeMethods that we will use in the project](https://github.com/microsoft/CsWin32#:~:text=Create%20a%20NativeMethods.txt%20file%20in%20your%20project%20directory%20that%20lists%20the%20APIs%20to%20generate%20code%20for.), in this instance CoCreateInstance for the COM used in FileDialogs

Reply:
As per latest commit, I've added the source generated code into App namespace. This file is now unnecessary and has been removed alongside the CsWin32 package.